src/checks/graphiteSpike.check.js

```diff
@@ -65,26 +66,38 @@ class GraphiteSpikeCheck extends Check {
 			fetch(this.sampleUrl)
 				.then(fetchres.json),
 			fetch(this.baselineUrl)
+				.then(fetchres.json),
+			fetch(this.recoveryUrl)
 				.then(fetchres.json)
 		])
 			.then(jsons => {
-
 				return this.normalize({
 					sample: jsons[0][0] ? jsons[0][0].datapoints[0][0] : 0,
 					// baseline should not be allowed to be smaller than one as it is use as a divisor
-					baseline: jsons[1][0] ? jsons[1][0].datapoints[0][0] : 1
+					baseline: jsons[1][0] ? jsons[1][0].datapoints[0][0] : 1,
+					recovery: jsons[2][0] ? jsons[2][0].datapoints[0][0] : 0
 				});
 			})
 			.then(data => {
 				let ok;
+				let recovery;
 				if (this.direction === 'up') {
 					ok = data.sample / data.baseline < this.threshold;
+					recovery = data.recovery / data.baseline < this.threshold;
```

Comment:
what if the sample period is half an hour, then this might mark it as resolved/not raise the alarm even if the error rate is consistently 6 times the norm